plugin/evm/log.go

```diff
@@ -75,7 +75,7 @@ func SubnetEVMJSONFormat(alias string) log.Format {
 			if !ok {
 				props[errorKey] = fmt.Sprintf("%+v is not a string key", r.Ctx[i])
 			} else {
-				props[k] = formatJSONValue(r.Ctx[i+1])
+				props[k] = formatJSONValue(r.Ctx[i+1]) // The number of arguments is normalized from the geth logger to prevent an index out of bounds here
```

Comment:
```suggestion
                                // The number of arguments is normalized from the geth logger
                                // to prevent an index out of bounds here
				props[k] = formatJSONValue(r.Ctx[i+1])
```

Reply:
uh hope this looks good

Reply:
copied same comment from coreth. hope this looks good.